### PR TITLE
DataPrep File Upload

### PR DESCRIPTION
**Pull request by elfenheart**
Opening this PR to unblock uploading file from local machine.

NOTE:
Will still have to implement final design and also limiting upload file size to 10 MB

Dependent on https://github.com/hydrator/wrangler/pull/88

### CODE REVIEW

File: cdap-ui/app/cdap/components/DataPrepConnections/UploadFile/index.js [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+  fileHandler(e) {
+    let stateObj = {
+      file: e[0]
+    };
+
+    if (e[0].size > FILE_SIZE_LIMIT) {
+      stateObj.error = T.translate(`${PREFIX}.fileSizeError`);
+    }
+
+    this.setState(stateObj);
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/DataPrepConnections/UploadFile/index.js]: on reupload of correct file it would be good if we hide the error message.

**Review by ajainarayanan — COMMENTED** [review]
Just one minor comment otherwise Code 👍 LGTM

**Review by ajainarayanan — APPROVED** [review]
👍 LGTM

**Review by bdmogal — APPROVED** [review]
LGTM.

Perhaps this also fixes https://issues.cask.co/browse/CDAP-9554 and https://issues.cask.co/browse/CDAP-11400?

**Pull request merged**